### ISSUE TITLE
Node can attempt to do GE even if not eligible

### ISSUE THREAD

**Issue by Derkades** [storj/storj#3905]
I can start a graceful exit without a warning that my node is not eligible. It won't actually do anything, running the `exit-status` command will show 0% completion forever. When the container is recreated, `exit-status` will display `No graceful exit in progress.` and the node prints an expected error message to stdout:
```
gracefulexit:chore      worker failed   {"error": "node is not yet eligible for graceful exit: will be eligible after ...
```

In my opinion there should be a clearer indication to the user which satellites they can exit.

**Comment by Derkades** [comment]
Did not test it but it should be fixed according to the 1.9.5 changelog

**Comment by ifraixedes** [comment]
@Derkades please if you try once your SN gets updated and it doesn't work as expected, make a comment and we'll reopen the issue.

Many thanks!
